apps/aechannel/src/aesc_force_progress_tx.erl

```diff
@@ -74,53 +58,13 @@
           offchain_trees:: aec_trees:trees(),
           ttl           :: aetx:tx_ttl(),
           fee           :: non_neg_integer(),
-          nonce         :: non_neg_integer(),
-          block_hash    :: aesc_pinned_block:hash()
```

Comment:
Is there a released version of the software with such record? I understand yes. So this breaks DB backward compatibility, that may be ok with 3.* though we need to decide this.

Reply:
Those records are allowed only from Fortuna on, so assumption is no such records on main net